ramls/edge-oai-pmh.raml

```diff
@@ -0,0 +1,97 @@
+#%RAML 1.0
+title: 'Edge API - OAI-PMH'
+baseUri: 'https://github.com/folio-org/edge-oai-pmh'
+version: v1
+documentation:
+  -
+    title: 'Edge API - OAI-PMH'
+    content: 'Edge API to interface with FOLIO for 3rd party harvesters to harvest metadta via OAI-PMH'
+types:
+    OAI-PMH: !include OAI-PMH.xsd
+traits:
+  resps:
```

Comment:
to improve readability of raml file I would propose to extract the traits to separate files and put them under ramls/traits directory

Reply:
I like this... good suggestion @epam-pavelk